bin/moveAndCheckSamplesheets.sh

```diff
@@ -334,10 +334,11 @@ fi
 for samplesheetChecked in "${samplesheetsChecked[@]}"
 do
 	# if samplesheets[@] is empty this means that the samplesheet is coming from a different machine, so we need logDir and a ${JOB_CONTROLE_FILE_BASE}.started file
```

Comment:
I do not understand the comment: why `we need logDir and a ${JOB_CONTROLE_FILE_BASE}.started file` when there are no samplesheets?

Reply:
`if samplesheets[@] is empty this means that the samplesheet is coming from a different machine`
This is wrong. if samplesheets[@] is empty this means that the samplesheet has been checked previously and was moved to the folder for checked samplesheets. That may have happened on a different machines, but it also may have happened during a previous execution of this script on this server.